src/arguments/CoreMember.ts

```diff
@@ -30,9 +30,9 @@ export class CoreArgument extends Argument<GuildMember> {
 			  });
 	}
 
-	private async resolveByID(argument: string, guild: Guild): Promise<GuildMember | null> {
-		const memberID = UserOrMemberMentionRegex.exec(argument) ?? SnowflakeRegex.exec(argument);
-		return memberID ? guild.members.fetch(memberID[1]).catch(() => null) : null;
+	private async resolveById(argument: string, guild: Guild): Promise<GuildMember | null> {
+		const memberId = UserOrMemberMentionRegex.exec(argument) ?? SnowflakeRegex.exec(argument);
+		return memberId ? guild.members.fetch(memberId[1] as Snowflake).catch(() => null) : null;
```

Comment:
```suggestion
		return memberId ? guild.members.fetch(memberId[1]).catch(() => null) : null;
```

The cast isn't neeeded